packages/terra-profile-image/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "terra-profile-image",
+  "main": "lib/ProfileImage.js",
+  "version": "0.1.0",
```

Comment:
When we do the initial release, the version will be bumped to 0.1.0. For now, please set this as 0.0.0